context/karma.js

```diff
@@ -84,6 +84,14 @@ var ContextKarma = function (callParentKarmaMethod) {
       self.log('alert', [msg])
     }
 
+    contextWindow.confirm = function (msg) {
+      self.log('confirm', [msg])
+    }
+
+    contextWindow.prompt = function (msg, defaultVal) {
+      self.log('prompt', [msg, defaultVal])
+    }
```

Comment:
I'm not sure, but I feel like we should call the original as well, because if a test relies on it that would fail now if I understand it correctly